### PR TITLE
Adding uilib snippet extension (not published)

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Adding uilib snippet extension (not published).
Waiting for https://github.com/wix/react-native-ui-lib/pull/1823 to generate new snippets.

## Changelog
Adding uilib snippet extension (not published)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "log": "react-native log-ios | grep 'ethan -'",
     "docs:deploy": "./scripts/deployDocs.sh",
     "docs:build": "node scripts/build-docs.js",
+    "build:snippets": "node scripts/generateSnippets.js",
```
Comment (ethanshar) [inline package.json:39]:
```suggestion
    "snippets:build": "node scripts/generateSnippets.js",
```

**Review by ethanshar — COMMENTED** [review]
Can we change the folder name into something a little shorter? maybe `rnuilib-snippets`? 
also, let's nest the whole folder under an `extensions` folder. 
Maybe we'll have more extensions in the future, and generally it'll be more structured

**Comment by Inbal-Tish** [comment]
> Can we change the folder name into something a little shorter? maybe `rnuilib-snippets`? also, let's nest the whole folder under an `extensions` folder. Maybe we'll have more extensions in the future, and generally it'll be more structured

Do you want to change the name of the extension or just the folder? I prefer to keep the folder and extension names the same to avoid confusion.

**Comment by ethanshar** [comment]
> > Can we change the folder name into something a little shorter? maybe `rnuilib-snippets`? also, let's nest the whole folder under an `extensions` folder. Maybe we'll have more extensions in the future, and generally it'll be more structured
> 
> Do you want to change the name of the extension or just the folder? I prefer to keep the folder and extension names the same to avoid confusion.

Change both the folder and the extension name. 
I try to brand ourselves as `rnuilib` (same as our twitter name) so I want to keep it consistent

**Comment by Inbal-Tish** [comment]
@ethanshar Done

**Review by ethanshar — APPROVED** [review]

**Pull request merged**